PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+## Description
```

Comment:
Maybe we can add `Fixes #...` so that the corresponding issue gets closed when PR is merged.

Reply:
We already have it below this. I think we can use that.

Reply:
I agree with @tshradheya that we should make the fixed issue clearer.
Ideally each pull request should have an issue it fixes.
So I would  be in favour of adding `Fixes #...` here and removing the "related issues and discussion" paragraph.

Reply:
@nicolas-raoul Sir I had updated the file please tell me if any other changes are needed.

Reply:
Sorry for the lack of clarity. I believe the "Fixes" was supposed to be **added** to the "Description" paragraph? So basically what is needed is 
```

## Description
Fixes #{Issue Number}
Describe the changes made and why they were made.
```